src/main/java/org/folio/inventory/dataimport/handlers/actions/InstanceUpdateDelegate.java

```diff
@@ -73,7 +74,15 @@ private JsonObject retrieveParsedContent(ParsedRecord parsedRecord) {
 
   private Future<Instance> getInstanceById(String instanceId, InstanceCollection instanceCollection) {
     Promise<Instance> promise = Promise.promise();
-    instanceCollection.findById(instanceId, success -> promise.complete(success.getResult()),
+    instanceCollection.findById(instanceId, success -> {
+        if (success.getResult() == null) {
+          String errorMsg = format("Can't find Instance by id: %s ", instanceId);
+          LOGGER.error(errorMsg);
```

Comment:
Maybe it would be better to use formating directly in Logger without setting new variable?

Reply:
@Abdulkhakimov Done.